usaspending_api/recipient/v2/views/list_recipients.py

```diff
@@ -86,10 +133,22 @@ def post(self, request):
         ]
         models.extend(copy.deepcopy(PAGINATION))  # page, limit, sort, order
 
+        # Override pagination default limit of 100
+        for model in models:
+            if model["name"] == "limit":
+                model["max"] = 1000
+
         new_sort = {"type": "enum", "enum_values": ["name", "duns", "amount"], "default": "amount"}
         models = update_model_in_list(models, "sort", new_sort)
         models = update_model_in_list(models, "limit", {"default": 50})
         validated_payload = TinyShield(models).block(request.data)
 
-        results, page_metadata = get_recipients(filters=validated_payload)
+        try:
+            count = self.request_count(validated_payload)
+        except Exception as e:
```

Comment:
How likely is this to fail? What would be the reasons for it to throw an exception?

Reply:
I think this is unlikely to fail. The try/except blocks are carry over from when this was making an actual http request, and I didn't think of removing them once I switched to directly calling the view. I think they can be removed.